src/NuGet.Core/NuGet.Common/Migrations/Migration1.cs

```diff
@@ -156,13 +157,13 @@ private static void FixPermissions(string path, PosixPermissions umask)
             {
                 PosixPermissions correctedPermissions = permissions.Value.WithUmask(umask);
                 string correctedPermissionsString = correctedPermissions.ToString();
-                Exec("chmod", correctedPermissionsString + " " + path);
+                Exec("chmod", correctedPermissionsString + " " + "\"" + path + "\"");
```

Comment:
This change hopefully addresses https://github.com/NuGet/NuGet.Client/commit/0966444b35d8a74fa25adf0368b983f389884377#r86528891 comment. I modified existing test to ensure path with spaces are working as expected.

Reply:
Do we have a precedent for this? 
Should we be using a built in escape method instead? 

This should be easy to test as a unit test as the escaping method is generic. 
We don't need a public API, an internal one would allow us to test it.

Reply:
Just making sure this doesn't get forgotten :)

Reply:
I pushed a new commit adding a new extension method to perform this string formatting. Please review and let me know the feedback.